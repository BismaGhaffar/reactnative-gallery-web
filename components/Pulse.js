import styled, { keyframes } from 'styled-components'
import { tada } from 'react-animations'

const pulser = keyframes`${tada}`

const Pulse = styled.div`
  animation: 1.5s ${pulser};
`

export default Pulse
