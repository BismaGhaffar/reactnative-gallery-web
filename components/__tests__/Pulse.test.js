import React from 'react'
import renderer from 'react-test-renderer'
import Pulse from '../Pulse'

it('Pulse can be created', () => {
  const comp = renderer.create(<Pulse />)
  expect(comp).toBeDefined()
})

it('<Pulse /> toMatchSnapshot', () => {
  const tree = renderer.create(<Pulse />).toJSON()
  expect(tree).toMatchSnapshot()
})
