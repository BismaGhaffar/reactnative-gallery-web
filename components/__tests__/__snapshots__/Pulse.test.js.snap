// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pulse /> toMatchSnapshot 1`] = `
<div
  className="Pulse-s1pfiyxk-0 fABGZs"
/>
`;
